lib/generated/intl/messages_en.dart

```diff
@@ -13,9 +13,10 @@
 import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 
-final messages = new MessageLookup();
+final messages = MessageLookup();
 
-typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
+typedef MessageIfAbsent = String Function(
+    String messageStr, List<dynamic> args);
```

Comment:
```suggestion
    String messageStr, 
    List<dynamic> args,
);
```

Reply:
Hmm agreed... I think I tried to convince the linter that was a good idea a while ago and the was some spiel about how they don't let you do that. Let me check again.

Reply:
Ah ok, thats apparently not part of dart format but part of dart fix... Not sure why and there also this open issue about something? https://github.com/dart-lang/dart_style/issues/1253...